modules/contract/contracts.js

```diff
@@ -238,9 +253,11 @@ class Contracts {
         gasLimit = gasLimit ? gasLimit : 500000;
       
         try {
+          const signer = await conseiljssoftsigner.SoftSigner.createSigner(conseiljs.TezosMessageUtils.writeKeyWithHint(keystore.privateKey,'edsk'), -1);
+
           const result = await conseiljs.TezosNodeWriter.sendContractInvocationOperation(
-                                      tezosNode, keystore, contractAddress, amount*1000000, fee, '', storageLimit, gasLimit, undefined, 
-                                      argument, conseiljs.TezosParameterFormat.Michelson);
+                                      tezosNode, signer, keystore, contractAddress, amount*1000000, fee, storageLimit, gasLimit, undefined, 
+                                      argument, conseiljs.TezosParameterFormat.Michelson, -1);
```

Comment:
why -1?

Reply:
it's counter. added in signer part in library.